tests/phpunit/tests/functions/wpUrlEncodeEndingPeriod.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * Tests for wp_url_encode_ending_period().
+ *
+ * @group  functions.php
+ * @covers ::wp_url_encode_ending_period
+ */
+class Tests_Functions_WpUrlEncodeEndingPeriod extends WP_UnitTestCase {
+
```

Comment:
May need to make sure the polyfills are loaded before this test runs.

```suggestion

	public static function set_up_before_class() {
		parent::set_up_before_class();

		// Make sure the polyfill is loaded before running this test.
		if ( ! function_exists( 'str_ends_with' ) ) {
			require_once ABSPATH . 'wp-includes/compat.php';
		}
	}
```

Reply:
Nope it's failing because this PR is set to `master` and not `trunk`. Will reopen a new PR and reference this one.